jvm/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -341,4 +341,8 @@ object ConverterUtils extends Logging {
   final val STARTS_WITH = "starts_with"
   final val SUBSTRING = "substring"
   final val ROW_CONSTRUCTOR = "row_constructor"
+
+  //
+  final val CHR = "chr"
+  final val ABS = "abs"
```

Comment:
Are these two functions already defined in Substrait yaml? You could check here: https://github.com/substrait-io/substrait/tree/main/extensions. If not, you can open PR to substrait to add.

Reply:
Have created a PR to  substrait-io/substrait